### PR TITLE
update some desc about auto-analyze

### PR DESCRIPTION
**Pull request by glkappe**
update some desc and format about auto-analyze, please

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: statistics.md [hunk 1]
```diff
@@ -314,9 +314,9 @@ ANALYZE INCREMENTAL TABLE TableName PARTITION PartitionNameList INDEX [IndexName
 
 ### 自动更新
 
-在发生增加，删除以及修改语句时，TiDB 会自动更新表的总行数以及修改的行数。这些信息会定期持久化下来，更新的周期是 20 * `stats-lease`，`stats-lease` 的默认值是 3s，如果将其指定为 0，那么将不会自动更新。
+在发生增加，删除以及修改语句时，TiDB 会自动更新表的总行数以及修改的行数。这些信息会定期持久化下来，更新的周期是 20 * [`stats-lease`](/tidb-configuration-file.md#stats-lease)。
```
Comment (TomShawn) [inline statistics.md:317]:
```suggestion
在发生增加，删除以及修改语句时，TiDB 会自动更新表的总行数以及修改的行数。这些信息会定期持久化下来，更新的周期是 20 * `stats-lease`。`stats-lease` 配置项的默认值是 3s，如果将其指定为 0，那么将不会自动更新，详细说明参见[TiDB 配置项文档 - `stats-lease`](/tidb-configuration-file.md#stats-lease)。
```

File: statistics.md [hunk 2]
```diff
@@ -314,9 +314,9 @@ ANALYZE INCREMENTAL TABLE TableName PARTITION PartitionNameList INDEX [IndexName
...
 
-和统计信息自动更新相关的三个系统变量如下：
+#### 系统变量
```
Comment (TomShawn) [inline statistics.md:319]:
```suggestion
#### 相关系统变量
```

File: statistics.md [hunk 3]
```diff
@@ -326,8 +326,12 @@ ANALYZE INCREMENTAL TABLE TableName PARTITION PartitionNameList INDEX [IndexName
 
 当某个表 `tbl` 的修改行数与总行数的比值大于 `tidb_auto_analyze_ratio`，并且当前时间在 `tidb_auto_analyze_start_time` 和 `tidb_auto_analyze_end_time` 之间时，TiDB 会在后台执行 `ANALYZE TABLE tbl` 语句自动更新这个表的统计信息。
 
+#### feedback
```
Comment (TomShawn) [inline statistics.md:329]:
```suggestion
#### 收集反馈信息

```

File: statistics.md [hunk 4]
```diff
@@ -326,8 +326,12 @@ ANALYZE INCREMENTAL TABLE TableName PARTITION PartitionNameList INDEX [IndexName
...
 在 TiDB v5.0 之前，执行查询语句时，TiDB 会以 [`feedback-probability`](/tidb-configuration-file.md#feedback-probability) 的概率收集反馈信息，并将其用于更新直方图和 Count-Min Sketch。**从 v5.0 起，该功能默认关闭，暂不建议开启此功能。**
 
+#### 是否开启自动更新
```
Comment (TomShawn) [inline statistics.md:332]:
```suggestion
#### 开启或关闭自动更新

```

File: statistics.md [hunk 5]
```diff
@@ -326,8 +326,12 @@ ANALYZE INCREMENTAL TABLE TableName PARTITION PartitionNameList INDEX [IndexName
...
+如果发现自动更新统计信息消耗过多的资源，影响在线业务，可以通过 [`run-auto-analyze`](/tidb-configuration-file.md#run-auto-analyze) 参数将其关闭
```
Comment (TomShawn) [inline statistics.md:333]:
```suggestion
如果发现自动更新统计信息消耗过多的资源，影响在线业务，可以通过 [`run-auto-analyze`](/tidb-configuration-file.md#run-auto-analyze) 参数关闭自动更新。
```

**Comment by qiancai** [comment]
Removed the **needs-cherry-pick-release-6.3** label because the v6.3 docs have been archived at https://docs-archive.pingcap.com/zh/tidb/v6.3 and will no longer receive new updates.

**Comment by lilin90** [comment]
@glkappe Would you please address the comments?

**Comment by qiancai** [comment]
Removed the needs-cherry-pick-release-6.4 label because the v6.4 docs have been archived at https://docs-archive.pingcap.com/zh/tidb/v6.4 and will no longer receive new updates.

**Review by qiancai — APPROVED** [review]

File: statistics.md [hunk 6]
```diff
@@ -344,6 +344,12 @@ ANALYZE INCREMENTAL TABLE TableName PARTITION PartitionNameList INDEX [IndexName
 
 为了避免小表因为少量数据修改而频繁触发自动更新，当表的行数小于 1000 时，TiDB 不会触发对此表的自动更新。你可以通过 `SHOW STATS_META` 来查看表的行数情况。
 
+#### 关闭自动更新
+
+如果发现自动更新统计信息消耗过多的资源，影响在线业务，可以通过 [`tidb_enable_auto_analyze`](/system-variables.md#tidb_enable_auto_analyze-从-v610-版本开始引入) 参数关闭自动更新。
```
Comment (qiancai) [inline statistics.md:349]:
```suggestion
如果发现自动更新统计信息消耗过多的资源，影响在线业务，可以通过系统变量 [`tidb_enable_auto_analyze`](/system-variables.md#tidb_enable_auto_analyze-从-v610-版本开始引入) 关闭自动更新。
```

**Pull request merged**